actor/kvcounter-wasmcon2023/wasmcloud.toml

```diff
@@ -5,3 +5,8 @@ type = "actor"
 [actor]
 claims = ["wasmcloud:httpserver", "wasmcloud:keyvalue"]
 wasm_target = "wasm32-wasi-preview2"
+wit_world = "kvcounter"
+# The local adapter file used in this repository
+# is required to use this demo since changes have already been made
+# upstream that make this adapter outdated.
+wasi_preview2_adapter_path = "wasi_snapshot_preview1.wasm"
```

Comment:
It would be nice to know where this adapter was sourced from, i.e. is there a stable URL you used?

Reply:
Oh yes, it's the 12.0 adapter, I'll add that in!